src/net.cpp

```diff
@@ -2492,6 +2492,9 @@ void DumpBanlist()
     CNode::GetBanned(banmap);
     bandb.Write(banmap);
 
+    // Notify UI about changed banlist
+    uiInterface.BannedListChanged();
```

Comment:
DumpBanlist() -> BannedListChanged() -> ClientModel::updateBanlist() -> banTableModel->refresh()... where are the multiple calls?